include/geos/algorithm/distance/Makefile.am

```diff
@@ -1,13 +1,14 @@
 #
-# This file is part of project GEOS (http://trac.osgeo.org/geos/) 
+# This file is part of project GEOS (http://trac.osgeo.org/geos/)
```

Comment:
Please don't mix style and functional changes (I know, can be annoying until we make all files respect .editorconfig, sorry about that)